.github/workflows/build-maven-plugin.yml

```diff
@@ -0,0 +1,38 @@
+name: build
+
+on: [push]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dangoslen/changelog-enforcer@v3
+  build:
+    name: Verify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 11
+          distribution: temurin
+
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: temurin
+
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: temurin
+
+      - name: Maven Verify
```

Comment:
The maven plugin has an e2e test with TeamServer where it uses credentials stored as secrets. The old credentials were Gilday's and are no longer usable. I think we should keep these tests but we do need to add new credentials to the repo in order to run them.

Reply:
we made a new test user and put the credentials in the secrets